ccp/src/prover/tests.rs

```diff
@@ -158,3 +255,102 @@ async fn prover_on_active_reschedule_on_active_commitment() {
 
     prover.stop().await.unwrap();
 }
+
+#[test(tokio::test(flavor = "multi_thread", worker_threads = 4))]
+async fn prover_on_active_extend_on_active_commitment_performance() {
```

Comment:
Would be awesome to manually check that directory is empty, b/c/ in future we can change the `get_proofs_after` logic to smth else.

Reply:
Cannot be implemented without race conditions.